CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v0.7.0 / 2020-07-23
+
+* [CHANGE] Switch logging to go-kit #73
+* [ENHANCEMENT] Add `memcached_time_seconds` metric #74
+* [ENHANCEMENT] Add slab metrics related to hot/warm/cold/temp LRUs #76
+* [BUGFIX] Fix `memcached_slab_mem_requested_bytes` metric in newer memcached versions #70
+* [BUGFIX] Fix `memcached_lru_crawler_starts_total` metric #83
```

Comment:
That's a change

Reply:
No, a change affects previously intended behavior. This never worked correctly and wasn't intended to be registered as `namespace_...`. Using change would not be in line how we have used the change lot categories in Prometheus before.

Reply:
In Prometheus metrics renames are considered changes

Reply:
Yes, if they were registered before correctly. Adding unregistered metrics which were supposed to be there are bugfixes. As this metric was not available under the memcached_ namespace due to the quoting issue, I consider it to be non-existing.

Reply:
OK

Reply:
@roidelapluie I made the change regardless. I hope that helps?

Reply:
Both are fine to me. Thanks!